### ISSUE TITLE
Invisible links on homepage are accessible via keyboard navigation (they shouldn't be) 

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#117]
Screengrab/videos (insert file names here)
Keyboard, Home page, Invisible Link.mp4

Link
https://d8-stg.boston.gov

Node # (if known)
21

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > macOS > Chrome 83

Trusted Tester Test
Keyboard

Brief Description
Keyboard can reach invisible links

Steps to reproduce
1)	00:00, Click in the URL field and start pressing the tab key until the focus enters the main page
2)	00:08, Note that the focus is not visible and chrome displayed the #skip-link anchor in the bottom left corner.
a)	BUG:  This link should not be accessible through the keyboard.
b)	Also visible at 00:20, 00:28, 00:33

Desired Result
Invisible links are skipped using the keyboard

Actual Result
An invisible link is reached using the keyboard
[Keyboard, Home page, Invisible Link.zip](https://github.com/CityOfBoston/Iterators/files/4983953/Keyboard.Home.page.Invisible.Link.zip)

**Comment by subaha-cob** [comment]
I had to redo a few things in drupal that was causing this. please test all elements and components carefully.

**Comment by subaha-cob** [comment]
Testing at: https://d8-uat.boston.gov/

**Comment by andrebtlima** [comment]
@subaha-cob still seeing this issue on UAT

**Comment by subaha-cob** [comment]
@andrebtlima So I have reduced this to just one invisible link. That is the one for skipping back to the main menu.

**Comment by subaha-cob** [comment]
@jill-iterators this is on the live site. So please test there unless I say otherwise. thanks.

**Comment by david-iterators** [comment]
This was retested on the live site and the skip-link is part of the tab order and shows up as an invisible link when navigating with the keyboard.  The following provides an updated bug report.  Note that the video is accessible through the link provided.

Screengrab/videos (insert file names here)
https://www.loom.com/share/d201737a3fe6456a9bc5b9df23e5dd00

Link
https://www.boston.gov

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 11.2.3 > Chrome 89

Trusted Tester Test
Visible Focus

Brief Description
The skip link is able to be tabbed to after the search button on the boston.gov site

Steps to reproduce
1)	00:00, Load the URL
2)	00:03, Start tabbing through the page
3)	00:11, Reach the Search button
4)	00:12, tab to the next element
a)	Bug: No visible focus is shown. The lower left hand corner of the page shows that the link tabbed to has a href of “https://boston.gov/#skip-link”

Desired Result
•	The skip link is not able to be tabbed to, and can only be accessed when the “Skip to Main Content” button is pressed

Actual Result
•	The skip link is able to be tabbed to

**Comment by subaha-cob** [comment]
@sebastianebarb We need a back to top button/icon for the website. We have a link for it currently, but it is hidden and the user can only navigate to it if using the tab button. I am not sure why it was implemented like that, but I need to fix it because it is part of my work fixing accessibility issues on the website. I can either remove it completely or add it properly, like an actual content that is visible for people to see. The content can be a text or image, it doesn't matter, but it needs to corrected.

**Comment by subaha-cob** [comment]
![image](https://user-images.githubusercontent.com/57374913/114581650-e26ad880-9c4d-11eb-8a4f-dc6e8b67c026.png)

![image](https://user-images.githubusercontent.com/57374913/114581672-e8f95000-9c4d-11eb-88f6-cb79ba0ede3e.png)

**Comment by subaha-cob** [comment]
@david-iterators I have updated UAT with changes for this ticket. Please test on UAT

**Comment by ollie-iterators** [comment]
This has been fixed
